extensions/amp-app-banner/0.1/amp-app-banner.js

```diff
@@ -397,6 +397,8 @@ export class AmpAndroidAppBanner extends AbstractAppBanner {
  */
 function handleDismiss(state) {
   hideBanner(state);
+  // Restore focus to document after dismiss for screen readers.
+  state.element.ownerDocument.body./*REVIEW*/focus();
```

Comment:
The focus is lost because of user interaction with the dismiss button and changing the focus to that button not the removal of the banner itself from the fixed layer. So it wouldn't make sense to focus on the document when we remove something from fixed layer programmatically it might actually break the user current focus.